.babelrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "shippedProposals": true,
+        "loose": true
+      }
+    ],
+    "@babel/preset-typescript"
+  ]
+}
```

Comment:
A habit I have gotten into after my last project here is that whenever there is some interesting config changes, or package version update, or something unintuitive, etc. - its good to leave a comment in here after you make the PR so that onlookers - reviewers or peers, can get an understanding of the motivations/impacts of the change.  Something as simple as "removed, unused" or "this allows us to do x [link to resource]".  I think we could get into that habit with this project.

So that being said, what is the purpose of this file?